### PR TITLE
First pass at launch files that run simulation *and* data collection

### PR DESCRIPTION
**Pull request by peasant98**
Per title.

### CODE REVIEW

File: scripts/data_generators/StaticPosesDataSaver.py [hunk 1]
```diff
@@ -232,7 +232,7 @@ def set_poses(self, time=3.0):
         """
         for pose in self.poses_list:
             positions, _, pose_name = pose[0], pose[1], pose[2]  # noqa: F841
-            self.controller.publish_positions(positions, 10)
+            self.controller.publish_positions(positions, 3)
```
Comment (watakandai) [inline scripts/data_generators/StaticPosesDataSaver.py]: I set this as `REST_TIME` in params file both for sim and real. i dont know why it's not merged ...

Comment (watakandai) [reply scripts/data_generators/StaticPosesDataSaver.py]: i'll add changes and push again.

**Review by watakandai — CHANGES_REQUESTED** [review]

File: scripts/data_generators/StaticPosesDataSaver.py (hunk 1 above)
Comment (peasant98) [reply scripts/data_generators/StaticPosesDataSaver.py]: sounds good

Comment (peasant98) [reply scripts/data_generators/StaticPosesDataSaver.py]: Any update?

**Review by watakandai — APPROVED** [review]

**Review by watakandai — APPROVED** [review]

**Comment by peasant98** [comment]
it's all working, folks!

**Pull request merged**